src/scripts.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const store = Redux.createStore(getNextState);
+
+const buttonUp = document.querySelector('.up');
+const buttonDown =document.querySelector('.down');
+
+buttonUp.addEventListener('click', () => {
+  store.dispatch(getMoveUpAction())
+});
+
+buttonDown.addEventListener('click', () => {
+  store.dispatch(getMoveDownAction())
+});
+
+function render() {
+  buttonUp.disabled = store.getState().selectedIndex === 0 || store.getState().selectedIndex === null;
+  buttonDown.disabled = store.getState().selectedIndex === store.getState().foodItems.length - 1 ||
+    store.getState().selectedIndex === null;
+
+  const foodList = document.querySelector('#food-list');
+  foodList.innerHTML = '';
+  const item = store.getState().foodItems;
+  item.map((i, index) => {
```

Comment:
use the correct method, the map should return new value and shouldn't have side effects